### ISSUE TITLE
不能复现结果

### ISSUE THREAD

**Issue by sunshine-ssl** [meteorshowers/RCF-pytorch#41]
您好，我用扩充过的BSDS500数据集训练，ODS最高到0.778，请问您训练的时候最大迭代次数和stepsize设置的是多少呢，我用BSDS500加PASCAL数据集训练的时候出来的是全灰的图，不知道是哪里有问题

**Comment by huangwefeng121** [comment]
> 您好，我用扩充过的BSDS500数据集训练，ODS最高到0.778，请问您训练的时候最大迭代次数和stepsize设置的是多少呢，我用BSDS500加PASCAL数据集训练的时候出来的是全灰的图，不知道是哪里有问题

请问怎么计算ODS和F-measure？

**Comment by zhujinchao123** [comment]
你们有没有用作者的模型直接测试？

**Comment by meteorshowers** [comment]
@sunshine-ssl   you can change the learning rate to fit the dataset to stable the training process
